src/components/SideBar.js

```diff
@@ -96,8 +96,14 @@ Menu.propTypes = {
 
 const ProfileMenu = ({ profileImage }) => {
   const { t } = useTranslation();
-  const [, , removeCookie] = useCookiesWithOptions();
+  const [cookies, , removeCookie] = useCookiesWithOptions(['user']);
   const router = useRouter();
+  const [username, setUsername] = useState('');
```

Comment:
Do we have to set it in the state? We can use the cookie value and default to an empty string, no?